packages/contracts-bedrock/src/L1/BatchInbox.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {NitroValidator} from "@nitro-validator/NitroValidator.sol";
+import {CborDecode} from "@nitro-validator/CborDecode.sol";
+import {CertManager} from "@nitro-validator/CertManager.sol";
+import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {ISemver} from "interfaces/universal/ISemver.sol";
+
+contract BatchInbox is OwnableUpgradeable, ISemver, NitroValidator {
+    /// @notice Semantic version.
+    /// @custom:semver 1.0.1
+    string public constant version = "1.0.1";
```

Comment:
I dont know why they use this, I dont think we need this

Reply:
I'm conforming to what OP does here in case we want to upstream in the future